<?xml version="1.0" encoding="utf-8"?><testsuite errors="1" failures="0" name="" skips="0" tests="0" time="0.690"><testcase classname="" name="scripts.FileUpload" time="0"><failure message="collection failure">/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/_pytest/python.py:353: in _importtestmodule
&gt;           mod = self.fspath.pyimport(ensuresyspath=True)
/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/py/_path/local.py:548: in pyimport
&gt;           __import__(modname)
/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/_pytest/assertion/rewrite.py:157: in load_module
&gt;           py.builtin.exec_(co, mod.__dict__)
scripts/FileUpload.py:3: in &lt;module&gt;
&gt;   from pages.upload import Upload
/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/_pytest/assertion/rewrite.py:157: in load_module
&gt;           py.builtin.exec_(co, mod.__dict__)
modules/pages/upload.py:1: in &lt;module&gt;
&gt;   from saunter.po.remotecontrol.page import Page
E     File &quot;/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/saunter/po/remotecontrol/page.py&quot;, line 23
E       import saunter.ConfigWrapper import ConfigWrapper
E                                         ^
E   SyntaxError: invalid syntax</failure></testcase></testsuite>